From dfe237e8fccb995e41b90bcb4ee479c5c5efb4b6 Mon Sep 17 00:00:00 2001
From: "jian.xu" <jian.xu@amlogic.com>
Date: Mon, 13 Oct 2014 21:59:29 +0800
Subject: [PATCH 5015/5965] PD#99290:PD#98324 :enlarge the apts record table to
 fix apts tooooo many ,overflow issue,which cause missing discontine

Change-Id: Ic98de1bb21122d20e3052e8a6c4019d2c0647104
---
 drivers/amlogic/amports/ptsserv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/amlogic/amports/ptsserv.c b/drivers/amlogic/amports/ptsserv.c
index eb8e6782f0fe..450779e5368c 100755
--- a/drivers/amlogic/amports/ptsserv.c
+++ b/drivers/amlogic/amports/ptsserv.c
@@ -13,7 +13,7 @@
 #include "vdec_reg.h"
 
 #define VIDEO_REC_SIZE  (8192*2)
-#define AUDIO_REC_SIZE  8192
+#define AUDIO_REC_SIZE  8192*2
 #define VIDEO_LOOKUP_RESOLUTION 2500
 #define AUDIO_LOOKUP_RESOLUTION 1024
 
-- 
2.19.0

